src/rules/no-separate-loading-state.ts

```
@@ -0,0 +1,136 @@
+          // Check if we have at least 2 elements (state and setter)
+          if (arrayPattern.elements.length >= 2) {
+            const stateElement = arrayPattern.elements[0];
+            const setterElement = arrayPattern.elements[1];
+
+            if (
+              stateElement?.type === AST_NODE_TYPES.Identifier &&
+              setterElement?.type === AST_NODE_TYPES.Identifier &&
+              isLoadingPattern(stateElement.name)
+            ) {
+              loadingStateVariables.set(stateElement.name, node);
+              setterUsages.set(setterElement.name, { truthy: false, falsy: false });
```

ESLint

[error] Replace `·truthy:·false,·falsy:·false` with `⏎················truthy:·false,⏎················falsy:·false,⏎·············`

(prettier/prettier)